dotnet/src/dotnetframework/GxClasses/Data/GXDataPostgreSQL.cs

```diff
@@ -76,7 +76,31 @@ public GxPostgreSql()
 			GXLogging.Debug(log, "Setting connectionString property ", ConnectionStringForLog);
 			return new PostgresqlConnectionWrapper(m_connectionString, connectionCache, isolationLevel);
 		}
+		string convertToSqlCall(string stmt, GxParameterCollection parameters)
+		{
+			if (parameters == null)
+				return stmt;
+			string pname;
+			StringBuilder sBld = new StringBuilder();
+			for (int i = 0; i < parameters.Count; i++)
+			{
+				if (parameters[i].Direction != ParameterDirection.Output && parameters[i].Direction!= ParameterDirection.ReturnValue)
+				{
+					if (sBld.Length>0)
+						sBld.Append(", ");
+					pname = ":" + parameters[i].ParameterName;
+					sBld.Append(pname);
+				}
+			}
+			return "SELECT " + stmt + "(" + sBld.ToString() + ")";
```

Comment:
I don't know if this case could happen...
'parameters' can be an EMPTY list? 
In this case, the returned statement would be an invalid one: "SELECT 'stmt' ()" ;

Reply:
@ggallotti yes, parameters can be empty. But it is the right way to call functions in PostgreSQL. That routine is only used to build calls to stored procedures in PostgreSQL.
Without the '()' it throws the error '42703: column "functionname" does not exist'
https://www.postgresql.org/docs/9.1/sql-createfunction.html